22-12-2020;18:47:14 WARN  o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration.openEntityManagerInViewInterceptor:220 - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
22-12-2020;18:47:16 WARN  c.n.c.s.URLConfigurationSource.<init>:121 - No URLs will be polled as dynamic configuration sources.
22-12-2020;18:47:16 WARN  c.n.c.s.URLConfigurationSource.<init>:121 - No URLs will be polled as dynamic configuration sources.
22-12-2020;18:47:17 WARN  o.s.c.l.c.BlockingLoadBalancerClientAutoConfiguration$BlockingLoadBalancerClientRibbonWarnLogger.logWarning:86 - You already have RibbonLoadBalancerClient on your classpath. It will be used by default. As Spring Cloud Ribbon is in maintenance mode. We recommend switching to BlockingLoadBalancerClient instead. In order to use it, set the value of `spring.cloud.loadbalancer.ribbon.enabled` to `false` or remove spring-cloud-starter-netflix-ribbon from your project.
22-12-2020;18:47:21 WARN  c.n.d.s.t.d.RetryableEurekaHttpClient.execute:130 - Request execution failed with message: java.net.ConnectException: Connection refused: no further information
22-12-2020;18:47:21 WARN  c.n.d.DiscoveryClient.get:303 - Using default backup registry implementation which does not do anything.
22-12-2020;18:47:22 WARN  c.n.d.DiscoveryClient.notify:1351 - Saw local status change event StatusChangeEvent [timestamp=1608659242108, current=DOWN, previous=UP]
22-12-2020;18:47:25 WARN  c.n.d.s.t.d.RetryableEurekaHttpClient.execute:130 - Request execution failed with message: java.net.ConnectException: Connection refused: no further information
22-12-2020;18:47:25 WARN  c.n.d.DiscoveryClient.register:875 - DiscoveryClient_UNICREDIT/host.docker.internal:unicredit:8091 - registration failed Cannot execute request on any known server
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:873)
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121)
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
22-12-2020;18:47:25 WARN  c.n.d.InstanceInfoReplicator.run:125 - There was a problem with the instance info replicator
com.netflix.discovery.shared.transport.TransportException: Cannot execute request on any known server
	at com.netflix.discovery.shared.transport.decorator.RetryableEurekaHttpClient.execute(RetryableEurekaHttpClient.java:112)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator$1.execute(EurekaHttpClientDecorator.java:59)
	at com.netflix.discovery.shared.transport.decorator.SessionedEurekaHttpClient.execute(SessionedEurekaHttpClient.java:77)
	at com.netflix.discovery.shared.transport.decorator.EurekaHttpClientDecorator.register(EurekaHttpClientDecorator.java:56)
	at com.netflix.discovery.DiscoveryClient.register(DiscoveryClient.java:873)
	at com.netflix.discovery.InstanceInfoReplicator.run(InstanceInfoReplicator.java:121)
	at com.netflix.discovery.InstanceInfoReplicator$1.run(InstanceInfoReplicator.java:101)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
22-12-2020;18:47:29 WARN  c.n.d.s.t.d.RetryableEurekaHttpClient.execute:130 - Request execution failed with message: java.net.ConnectException: Connection refused: no further information
22-12-2020;22:22:08 WARN  o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration.openEntityManagerInViewInterceptor:220 - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - relation "account" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - relation "customer_account" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - relation "merchant_order" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - relation "payment_request" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - relation "payment_request" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - relation "seller_account" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - relation "transaction" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - relation "transaction" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - table "account" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - table "address" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - table "customer_account" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - table "merchant_order" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - table "order_counter" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - table "payment_request" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - table "seller_account" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - table "transaction" does not exist, skipping
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:230 - SQL Warning Code: 0, SQLState: 00000
22-12-2020;22:22:09 WARN  o.h.e.j.s.SqlExceptionHelper.logWarning:231 - table "url_responder" does not exist, skipping
22-12-2020;22:22:10 WARN  c.n.c.s.URLConfigurationSource.<init>:121 - No URLs will be polled as dynamic configuration sources.
22-12-2020;22:22:10 WARN  c.n.c.s.URLConfigurationSource.<init>:121 - No URLs will be polled as dynamic configuration sources.
22-12-2020;22:22:10 WARN  o.s.c.l.c.BlockingLoadBalancerClientAutoConfiguration$BlockingLoadBalancerClientRibbonWarnLogger.logWarning:86 - You already have RibbonLoadBalancerClient on your classpath. It will be used by default. As Spring Cloud Ribbon is in maintenance mode. We recommend switching to BlockingLoadBalancerClient instead. In order to use it, set the value of `spring.cloud.loadbalancer.ribbon.enabled` to `false` or remove spring-cloud-starter-netflix-ribbon from your project.
22-12-2020;22:27:33 WARN  o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration.openEntityManagerInViewInterceptor:220 - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
22-12-2020;22:27:34 WARN  c.n.c.s.URLConfigurationSource.<init>:121 - No URLs will be polled as dynamic configuration sources.
22-12-2020;22:27:34 WARN  c.n.c.s.URLConfigurationSource.<init>:121 - No URLs will be polled as dynamic configuration sources.
22-12-2020;22:27:34 WARN  o.s.c.l.c.BlockingLoadBalancerClientAutoConfiguration$BlockingLoadBalancerClientRibbonWarnLogger.logWarning:86 - You already have RibbonLoadBalancerClient on your classpath. It will be used by default. As Spring Cloud Ribbon is in maintenance mode. We recommend switching to BlockingLoadBalancerClient instead. In order to use it, set the value of `spring.cloud.loadbalancer.ribbon.enabled` to `false` or remove spring-cloud-starter-netflix-ribbon from your project.
22-12-2020;22:31:59 WARN  o.s.b.a.o.j.JpaBaseConfiguration$JpaWebConfiguration.openEntityManagerInViewInterceptor:220 - spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
22-12-2020;22:32:01 WARN  c.n.c.s.URLConfigurationSource.<init>:121 - No URLs will be polled as dynamic configuration sources.
22-12-2020;22:32:01 WARN  c.n.c.s.URLConfigurationSource.<init>:121 - No URLs will be polled as dynamic configuration sources.
22-12-2020;22:32:01 WARN  o.s.c.l.c.BlockingLoadBalancerClientAutoConfiguration$BlockingLoadBalancerClientRibbonWarnLogger.logWarning:86 - You already have RibbonLoadBalancerClient on your classpath. It will be used by default. As Spring Cloud Ribbon is in maintenance mode. We recommend switching to BlockingLoadBalancerClient instead. In order to use it, set the value of `spring.cloud.loadbalancer.ribbon.enabled` to `false` or remove spring-cloud-starter-netflix-ribbon from your project.
22-12-2020;22:32:02 WARN  c.n.d.DiscoveryClient.notify:1351 - Saw local status change event StatusChangeEvent [timestamp=1608672722523, current=DOWN, previous=UP]
